### DISCUSSION TITLE
Where to get the config command from?

### DISCUSSION THREAD

**Discussion by txtyash** [nushell/nushell#5225] [Q&A] [answered]
I installed nushell from arch's official repo but seems like config command wasn't installed.

Also how do I change the vim mode shortcuts?

**Comment by kubouch** [comment, marked as answer]
Hi, the config command is gone, we've now transferred to using `config.nu`. See https://www.nushell.sh/book/configuration.html . (We're now experimenting moving the current `config` variable to be an environment variable -- this might be for the next release.)

I think there is no way to change the basic vi mode keys (h j k l, ...). Some of the vi mode functionality is not yet complete (not all vi moves/actions are supported yet).

However, you can define custom keybindings within the config. Here is an example from the default config: https://github.com/nushell/nushell/blob/3783c19d02f4a3b2f35cd26b79daabcb70b04880/docs/sample_config/default_config.nu#L266 . Also check the `keybindings` command, it has useful subcommands that should help you define your own keybindings.

**Reply by txtyash** [reply]
> Hi, the config command is gone, we've now transferred to using `config.nu`. See https://www.nushell.sh/book/configuration.html . (We're now experimenting moving the current `config` variable to be an environment variable -- this might be for the next release.)

Please update it in README.md
